Windows format tests

```
[ERROR] Failures: 
[ERROR]   MilanVisitorTest.visitsNotSimpleWhileStatement:155 Outputs 15 times
Expected: "1\n2\n2\n3\n3\n3\n4\n4\n4\n4\n5\n5\n5\n5\n5\n"
     but: was "1\r\n2\r\n2\r\n3\r\n3\r\n3\r\n4\r\n4\r\n4\r\n4\r\n5\r\n5\r\n5\r\n5\r\n5\r\n"
[ERROR]   MilanVisitorTest.visitsOutput:62 Write right output
Expected: "101\n"
     but: was "101\r\n" 
```

--- a/milan-interpreter/pom.xml
+++ b/milan-interpreter/pom.xml
@@ -52,6 +52,10 @@ SOFTWARE.
       <artifactId>cactoos</artifactId>
     </dependency>
     <dependency>
+      <groupId>ru.l3r8y</groupId>
+      <artifactId>unixized</artifactId>
+    </dependency>
+    <dependency>
       <groupId>org.junit.jupiter</groupId>
       <artifactId>junit-jupiter-api</artifactId>
     </dependency>

--- a/milan-interpreter/src/test/java/ru/milan/interpreter/fake/FakeLexer.java
+++ b/milan-interpreter/src/test/java/ru/milan/interpreter/fake/FakeLexer.java
@@ -2,11 +2,17 @@ package ru.milan.interpreter.fake;
 
 import org.antlr.v4.runtime.CharStreams;
 import org.cactoos.io.ResourceOf;
+import ru.l3r8y.UnixizedOf;
 import ru.milan.interpreter.MilanLexer;
 
 public class FakeLexer extends MilanLexer {
 
     public FakeLexer(final String resource) throws Exception {
-        super(CharStreams.fromStream(new ResourceOf(resource).stream()));
+        super(CharStreams.fromString(
+            new UnixizedOf(new ResourceOf(resource))
+                .asText()
+                .asString()
+            )
+        );
     }
 }

--- a/pom.xml
+++ b/pom.xml
@@ -41,6 +41,7 @@ SOFTWARE.
     <mockito-core.version>5.2.0</mockito-core.version>
     <junit-pioneer.version>2.0.1</junit-pioneer.version>
     <picocli.version>4.7.1</picocli.version>
+    <unixized.version>1.0.0</unixized.version>
   </properties>
   <packaging>pom</packaging>
   <modules>
@@ -94,6 +95,11 @@ SOFTWARE.
         <groupId>org.projectlombok</groupId>
         <artifactId>lombok</artifactId>
         <version>${lombok.version}</version>
+      </dependency>
+      <dependency>
+        <groupId>ru.l3r8y</groupId>
+        <artifactId>unixized</artifactId>
+        <version>${unixized.version}</version>
       </dependency>
       <dependency>
         <groupId>org.cactoos</groupId>

